_posts/2023-04-24-adopting-apache-arrow-at-cloudquery.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Adopting Apache Arrow at CloudQuery"
+date: "2023-04-26 00:00:00"
```

Comment:
Before this PR is merged, please update this date to the actual publish date and rename the file accordingly.

Reply:
I don't have merge permissions but updated for today. mind merging it in ? :)

Reply:
Hi @yevgenypats  -- thank you for the update. Given the discussion on the mailing list is only a few days old I recommend we wait a few more days to see if there are additional comments.

https://lists.apache.org/thread/mbbx6xzhp9zynkvncpw68j9f71p6gsbp

I can handle merging this (and updating the date) when it is ready, perhaps sometime mid week